packages/cinterop/build.gradle.kts

```diff
@@ -82,12 +84,26 @@ android {
     }
 }
 
-val nativeLibraryIncludes = mutableListOf(
-    "-include-binary", "$rootDir/../external/core/build-macos_x64/src/realm/object-store/c_api/librealm-ffi-static-dbg.a",
-    "-include-binary", "$rootDir/../external/core/build-macos_x64/src/realm/librealm-dbg.a",
-    "-include-binary", "$rootDir/../external/core/build-macos_x64/src/realm/parser/librealm-parser-dbg.a",
-    "-include-binary", "$rootDir/../external/core/build-macos_x64/src/realm/object-store/librealm-object-store-dbg.a"
+val nativeLibraryIncludesMacosUniversal = listOf(
```

Comment:
Looks like something that could be defined as a list and mapped onto a specific base platform directory.

Reply:
I don't think we can use the `listOf().map{}` transformation since the ` "-include-binary"` needs to be emitted as a sperate argument (not concatenated with space `"-include-binary $rootDir/../external/core/$it"`) otherwise it won't work